voxel51/query.py

```diff
@@ -246,4 +246,5 @@ def __init__(self):
         '''Initializes a JobsQuery instance.'''
         super(JobsQuery, self).__init__([
             "id", "name", "state", "archived", "upload_date", "analytic_id",
-            "auto_start", "use_gpu"])
+            "auto_start", "use_gpu", "start_date", "completion_date",
```

Comment:
I just read documentation for `queued_time ` and `execution_time` in the `platform-docs`. We need to be more careful about keeping API + client libs + docs in-sync.

Reply:
That is available on the job-statistics query, not the standard `v1/jobs` query (which this is).

Reply:
Hmm, okay.